frontend/src/metabase/static-viz/components/XYChart/types.ts

```diff
@@ -15,13 +16,28 @@ export type YAxisPosition = "left" | "right";
 
 export type VisualizationType = "line" | "area" | "bar" | "waterfall";
 
-export type Series = {
-  name: string;
-  color: string;
+interface BaseSeries {
+  name: string | null;
   data: SeriesData;
   type: VisualizationType;
   yAxisPosition: YAxisPosition;
-};
+}
+
+export interface SeriesWithOneOrLessDimensions extends BaseSeries {
+  cardName: string;
+  column: DatasetColumn;
+}
+
+export interface SeriesWithTwoDimensions extends BaseSeries {
+  cardName: string;
+  column: DatasetColumn;
+  breakoutValue: string;
+}
+
+export interface Series extends BaseSeries {
+  name: string;
+  color: string;
+}
```

Comment:
This type is still the same, and is used for rendering static Line/Area/Bar chart.